roles/install_octoprint/files/config_recore.yaml

```diff
@@ -0,0 +1,118 @@
+appearance:
+  name: Recore
+folder:
+  uploads: /usr/share/models
```

Comment:
This is non-standard and messes with the file permissions.
Normally the path should be `/home/debian/.octoprint/uploads/` instead.